partition/roles/ztp/tasks/main.yaml

```diff
@@ -26,11 +26,14 @@
     - ztp.json.j2
     - user.sh.j2
 
-- name: copy frr.conf
+- name: copy frr.conf and config_db.json
   copy:
-    dest: "{{ ztp_host_dir_path }}/config/frr.conf"
-    src: frr.conf
+    dest: "{{ ztp_host_dir_path }}/config/{{ item }}"
+    src: "{{ item }}"
     mode: 0644
+  loop:
+    - frr.conf
+    - config_db.json
```

Comment:
Before this, the role was agnostic in terms of switch operating system, right? Does it make sense to have this file Cumulus?

Reply:
No it wasn't agnostic before. There is a `ztp.json` file and a `ztp_provisioning_script` variable which are only relevant for SONiC. Also the `frr.conf` isn't used by Cumulus deployments.